sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -11,38 +11,39 @@
         application.
       </p>
       <v-row>
+        <v-col>
+          <v-text-field
+            density="compact"
+            label="Application Number"
```

Comment:
I believe that all labels should follow the case like `Application number` and the validations like `Invalid application number`.